projects/plugins/jetpack/modules/sso.php

```diff
@@ -81,6 +82,128 @@ private function __construct() {
 			require_once JETPACK__PLUGIN_DIR . 'modules/sso/class-jetpack-force-2fa.php';
 			new Jetpack_Force_2FA();
 		}
+		add_filter( 'manage_users_columns', array( $this, 'jetpack_user_connected_th' ) );
+		add_action( 'admin_print_styles-users.php', array( $this, 'jetpack_user_table_styles' ) );
+		add_action( 'manage_users_custom_column', array( $this, 'jetpack_show_connection_status' ), 10, 3 );
+		add_action( 'admin_post_jetpack_invite_user_to_wpcom', array( $this, 'invite_user_to_wpcom' ) );
+		add_action( 'admin_notices', array( $this, 'handle_invitation_results' ) );
+	}
+
+	/**
+	 * Renders invitations errors/success messages in users.php.
+	 */
+	public function handle_invitation_results() {
+		$valid_nonce = isset( $_GET['_wpnonce'] ) ? wp_verify_nonce( $_GET['_wpnonce'], 'jetpack-sso-invite-user' ) : false; // phpcs:ignore WordPress.Security.ValidatedSanitizedInput -- WP core doesn't pre-sanitize nonces either.
+
+		if ( ! $valid_nonce || ! isset( $_GET['jetpack-sso-invite-user'] ) ) {
+			return;
+		}
+		if ( $_GET['jetpack-sso-invite-user'] === 'success' ) {
+			return wp_admin_notice( __( 'User was invited successfully!', 'jetpack' ), array( 'type' => 'success' ) );
+		}
+		if ( $_GET['jetpack-sso-invite-user'] === 'failed' && isset( $_GET['jetpack-sso-invite-error'] ) ) {
+			switch ( $_GET['jetpack-sso-invite-error'] ) {
+				case 'invalid-user':
+					return wp_admin_notice( __( 'Tried to invite a user that either doesn&#8217;t exist.', 'jetpack' ), array( 'type' => 'error' ) );
```

Comment:
Can we update this to : `Tried to invite a user that doesn&#8217;t exist.`?
Only need to remove the word `either` from the sentence.

Reply:
Fixed